src/main/resources/liquibase/1.0/changelog-1.0.0.yml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - logicalFilePath: liquibase/1.0/changelog-1.0.0.yml
+  - changeSet:
+      id: created subscribers table
+      author: Ali Huseynov
+      changes:
+        - createTable:
+            tableName: subscribers
+            columns:
+              - column:
+                  name: id
+                  type: serial
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: email
+                  type: varchar(70)
```

Comment:
You did not research enough about the email length validity : 
https://stackoverflow.com/questions/386294/what-is-the-maximum-length-of-a-valid-email-address